job_templates/snippet/property_parameter-definition.xml.em

```diff
@@ -42,5 +42,27 @@ This tests the robustness to whitespace being within the different paths.</descr
           <defaultValue>@(test_args_default)</defaultValue>
           <trim>false</trim>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.ChoiceParameterDefinition>
+          <name>CI_ENABLE_SANITIZER_TYPE</name>
+          <description>This can be used to enable sanitizer based builds and tests</description>
+          <choices class="java.util.Arrays$ArrayList">
+            <a class="string-array">
+              <string>@enable_sanitizer_type_default</string>
```

Comment:
not using a macro would be more clear IMHO
Here, you put everything in the vector, just to remove one, just to print a single value => https://en.wikipedia.org/wiki/YAGNI

Reply:
Makes sense, I have simplified it a bit by removing the for loop. Will add it back in the PR for tsan job

Reply:
Makes sense, I have simplified it a bit by avoiding the for loop